doc/man3/BIO_printf.pod

```diff
@@ -38,6 +38,11 @@ All functions return the number of bytes written, or -1 on error.
 For BIO_snprintf() and BIO_vsnprintf() this includes when the output
 buffer is too small.
 
+=head1 NOTES
+
+Both BIO_snprintf() and BIO_vsnprintf() null terminate their output when there
+is insufficient space.
```

Comment:
Maybe: `Both BIO_snprintf() and BIO_vsnprintf() terminate their output with B<NUL> even in case there is insufficient space to output the whole string with the obvious exception of I<n> being 0.`

Reply:
I've tweaked the wording a bit...